packages/wrangler/src/update-check.ts

```
@@ -0,0 +1,22 @@
+import chalk from "chalk";
+import checkForUpdate from "update-check";
+import pkg from "../package.json";
+
+export async function updateCheck(): Promise<JSX.Element | null> {
+  let update = null;
+  try {
+    // default cache for update check is 1 day
+    update = await checkForUpdate(pkg, {
+      distTag: "beta",
+    });
+  } catch (err) {
+    console.error(`Failed to check for updates: ${err}`);
```

This should probably be a silent failure